__tests__/commands/workspaces.js

```diff
@@ -42,3 +47,17 @@ test('workspaces info should list the workspaces', (): Promise<void> => {
     });
   });
 });
+
+test('workspaces run should spawn command for each workspace', (): Promise<void> => {
```

Comment:
@arcanis

I thought test would cover the passing of the arguments, or am I missing something. It sounds like I may need to either leave `run` in the arg list or add an additional `--` for the user?

Reply:
The problem with those tests is that when you call `runWorkspaces`, it directly invokes the command, without going through our CLI code. If it did, you would see that the `--flag1` flag would be interpreted as a command-line option for the `yarn workspaces run` command, and wouldn't actually be passed to the argument array.

Reply:
@arcanis Can you confirm if my additional testing steps are adequate?